src/variables/body/useTbody.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Jupyter Development Team.
```

Comment:
Is this function used in multiple places? If not, it should probably just be defined in the file where it's used. I'm not sure I understand why it's not in-line where you call it.